scripts/scripts/japanese_to_romaji.py

```
@@ -16,20 +16,77 @@
-1. Use standard Hepburn romanization.
-2. Keep the original punctuation and spacing.
-3. Do not translate the text, only convert it to Romaji.
-4. Output just the converted text, nothing else.
-5. Capitalize correctly.
-6. Use macrons and apostrophes where appropriate.
-
-Example Input: こんにちは
-Example Output: Konnichiwa
-
-Input: """
+SYSTEM_PROMPT = """Convert Japanese text to Romaji using the Modified Hepburn system with meticulous attention to detail. Adhere strictly to the rules below:
```

Pylint (3.3.7)

[convention] Line too long (157/100)

(C0301)